docs/gym-skill.md

```diff
@@ -0,0 +1,79 @@
+The AEA Gym Skill demonstrates how a custom Reinforcement Learning agent may be embedded into an Autonomous Economic Agent.
+
+
+## Demo instructions
+
+Make sure you have followed the Quick Start setup instructions <a href="../quickstart" target=_blank>here</a>, including downloading the `examples` and `scripts` directories.
+
+Then, download the channels directory.
```

Comment:
They no longer need to to that, instead
`svn export https://github.com/fetchai/agents-aea.git/trunk/examples`